site-content/source/modules/ROOT/pages/blog.adoc

```diff
@@ -8,6 +8,31 @@ NOTES FOR CONTENT CREATORS
 - Replace post tile, date, description and link to you post.
 ////
 
+//start card
+[openblock,card shadow relative test]
+----
+[openblock,card-header]
+------
+[discrete]
+=== Apache Cassandra Changelog #15
+[discrete]
+==== May 5, 2022
+------
+[openblock,card-content]
+------
+The project hits the code freeze button in readiness for the release of Apache Cassandra 4.1, more volunteers are needed for the Build Lead role, and CFPs are open for the Cassandra and Performance Engineering tracks at ApacheCon NA.
```

Comment:
@dtopdontstop @cthornett , I'm editing the teaser to make it fit inside the button:
```suggestion
Code freeze in readiness for Cassandra 4.1 release, volunteers needed for Build Lead role, and CFPs open for Cassandra and Performance Engineering tracks at ApacheCon NA.
```